pyanaconda/screen_access.py

```diff
@@ -0,0 +1,220 @@
+                self._config.write(f)
+
+    def _get_new_config_header(self):
+        """Generate a header for use when Anaconda generates the user interaction config file.
+
+        :returns: an appropriate user config file header
+        :rtype: str
+        """
+        return "# This file has been generated by the Anaconda Installer %s\n\n" % startup_utils.get_anaconda_version_string()
+
+    def _parse_bool_option(self, section_name, option_name):
+        """Read a boolean option from the user config file.
+
+        Make sure to convert the value used for boolean values (1 & 0) to Python booleans (True & False)
+        and throw SyntaxError if an invalid value is specified and return None if the option has not been found.
+
+        :param str section_name: section where to look for the option
+        :param str option_name: name of the option to check
+        :returns: True/False for 1/0 and None id the option has not been found in the section
+        :rtype: bool or None
+        """
+        option_value = self._config.get(section_name, option_name, fallback=None)
+        if option_value is None:
+            return option_value
+        # the option has some content
+        option_boolean = CONFIG_BOOLEAN_TABLE.get(option_value)
+        if option_boolean is None:
+            # parsing failed for the option
+            raise SyntaxError
+        else:
+            return option_boolean
+
+    @property
+    def post_install_tools_disabled(self):
+        """Report if post install tools should be marked as disabled in the user interaction config file."""
+        with self._lock:
+            if self._config.has_section(CONFIG_GENERAL_SECTION):
+                try:
+                    # convert to bool in case the option is not set (the call returns None)
+                    return bool(self._parse_bool_option(CONFIG_GENERAL_SECTION, CONFIG_DISABLE_POSTINST_TOOLS_KEY))
+                except SyntaxError:
+                    log.error("Can't check if post install tools have been disabled due to config file syntax error.")
+            else:
+                return False
+
+    @post_install_tools_disabled.setter
+    def post_install_tools_disabled(self, tools_disabled):
+        """Controls if post install tools should be marked as disabled in the user interaction config file."""
+        with self._lock:
+            if not self._config.has_section(CONFIG_GENERAL_SECTION):
+                self._config.add_section(CONFIG_GENERAL_SECTION)
+            if tools_disabled:
+                self._config.set(CONFIG_GENERAL_SECTION, CONFIG_DISABLE_POSTINST_TOOLS_KEY, CONFIG_TRUE)
+            else:
+                self._config.set(CONFIG_GENERAL_SECTION, CONFIG_DISABLE_POSTINST_TOOLS_KEY, CONFIG_FALSE)
+
+    def mark_screen_visited(self, screen_name):
+        """Mark the given screen as visited
+
+        :param str screen_name: screen to mark as visited
+        """
+
```

Comment:
Please be consistent with the new lines between docstring comments and code. There is one line somewhere and nothing elsewhere.